execution_engine_testing/tests/src/test/contract_api/runtime.rs

```diff
@@ -123,8 +123,8 @@ fn should_hash() {
 
         builder.exec(exec_request).commit().expect_success();
 
-        let digest = get_value::<BLAKE2B_DIGEST_LENGTH>(&builder, HASH_RESULT);
-        let expected_digest = crypto::blake2b(input);
+        let digest = get_value::<{ cryptography::DIGEST_LENGTH }>(&builder, HASH_RESULT);
```

Comment:
Unnecessary braces?

Reply:
[RFC2000](https://rust-lang.github.io/rfcs/2000-const-generics.html#applying-a-const-as-a-parameter)

> When applying an expression as const parameter (except for arrays), which is not an identity expression, the expression must be contained within a block. This syntactic restriction is necessary to avoid requiring infinite lookahead when parsing an expression inside of a type.

`cryptography::DIGEST_LENGTH` is not an identity expression, and it won't compile. `DIGEST_LENGTH` alone is, so it's easily fixable